src/utils/storageUtils.ts

```diff
@@ -0,0 +1,22 @@
+const storage = {
+    // returns the keyValue if it exists else returns null
+    getItem: <TTokenValue>(keyName: string) => {
+        const tokenVal = localStorage.getItem(keyName)
+            ? (JSON.parse(
+                  localStorage.getItem(keyName) as string
```

Comment:
Can't you write this without "as string". Is this necessary?

Reply:
Removed the type assertion.